Missing http gcrypt extension

### Version of install-php-extensions

2.2.17

### Error description

The `http` extension fails to install.  It tries to include `gcrypt.h` during compiling, which isn't present.

```
30.54 /tmp/pear/temp/pecl_http/src/php_http_curl.c:28:33: fatal error: gcrypt.h: No such file or directory
30.54    28 | #                       include <gcrypt.h>
30.54       |                                 ^~~~~~~~~~
30.54 compilation terminated.
```

### Docker image

php:8.3.8-zts

### Minimal Dockerfile

```Dockerfile
FROM php:8.3.8-zts

# Copy PHP extension install script
COPY --link --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/

# Install the needed PHP extensions
RUN set -eux; install-php-extensions http;
```

--- a/install-php-extensions
+++ b/install-php-extensions
@@ -952,7 +952,7 @@ buildRequiredPackageLists() {
 					buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile ^libidn1[0-9+]-dev$"
 				else
 					buildRequiredPackageLists_persistent="$buildRequiredPackageLists_persistent ^libicu[0-9]+$ ^libidn2-[0-9+]$"
-					buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile libicu-dev ^libidn2-[0-9+]-dev$"
+					buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile libicu-dev ^libidn2-[0-9+]-dev$ ^libgcrypt[0-9]+-dev$"
 				fi
 				;;
 			imagick@alpine)

